AsyncDisplayKit/ASMapNode.mm

```diff
@@ -24,7 +24,7 @@ @interface ASMapNode()
   MKMapSnapshotter *_snapshotter;
   BOOL _snapshotAfterLayout;
   NSArray *_annotations;
-  CLLocationCoordinate2D _centerCoordinateOfMap;
+//  CLLocationCoordinate2D _centerCoordinateOfMap;
```

Comment:
If this is not needed anymore, could you delete it instead of leaving it commented out?  Same below.